src/tracing/mux.rs

```diff
@@ -322,6 +323,10 @@ pub enum Error {
     #[error("expected config is missing for tracer '{0:?}'")]
     MissingConfig(GethDebugBuiltInTracerType),
     /// Error when deserializing the config
+    #[cfg(feature = "std")]
     #[error("error deserializing config: {0}")]
```

Comment:
I believe this type works in no-std as long as the serde-json alloc feature is enabled